auth/services/oauth/holder.go

```diff
@@ -102,14 +106,20 @@ func (v *HolderService) ClientMetadata(ctx context.Context, endpoint string) (*o
 	return metadata, nil
 }
 
-func (v *HolderService) PostError(ctx context.Context, auth2Error oauth.OAuth2Error, verifierResponseURI string) (string, error) {
+func (v *HolderService) PostError(ctx context.Context, auth2Error oauth.OAuth2Error, verifierResponseURI string, verifierClientState string) (string, error) {
```

Comment:
`verifier` is superfluous in `verifierClientState` here; in this flow the wallet is the AS and verifier the RP (thus client), thus `clientState` can only be the other party's.

Reply:
that's why it's called the `verifierClientState`, as in: the client state of the verifier who has the RP role in this part.

Reply:
You're right, it's defined as just `state` (https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.1). But a parameter `state` still always refers to the client/RP. But it's so local here, it doesn't really matter.